pkg/statistics/region.go

```diff
@@ -59,7 +59,7 @@ func (s *RegionStats) Observe(r *core.RegionInfo) {
 	approximateKeys := r.GetApproximateKeys()
 	approximateSize := r.GetApproximateSize()
 	approximateKvSize := r.GetApproximateKvSize()
-	if approximateSize <= core.EmptyRegionApproximateSize {
+	if approximateSize == core.EmptyRegionApproximateSize {
 		s.EmptyCount++
 	}
 	s.StorageSize += approximateSize
```

Comment:
approximateSize == 1 means empty, should we add it to storage size?

Reply:
Do u mean current storage size? If so, it is calculated from the used size of store

Reply:
I mean 
```go
if approximateSize > core.EmptyRegionApproximateSize {
   s.StorageSize += approximateSize
}
```